### PR TITLE
button lib

### PR DESCRIPTION
**Pull request by yelendisureshbabu-hcl**
created button library

### CODE REVIEW

File: projects/buttons/src/lib/button.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+@Component({
+  selector: '[xotbButton]',
+  template: `
+    <xotb-styled-component>
+      <svg
+        class="xotb-button__icon xotb-button__icon_left"
+        *ngIf="hasLeftIcon()"
+        [xotbIconName]="iconName"
+      ></svg>
```
Comment (akuntal) [inline projects/buttons/src/lib/button.ts]: Put that html template in separate html file.

File: projects/buttons/src/public-api.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+/*
+ * Public API Surface of buttons
+ */
+
+export * from './lib/button';
```
Comment (akuntal) [inline projects/buttons/src/public-api.ts]: don't export button component, only module should be fine

File: projects/xotb-demo/src/app/examples/buttons/buttons.component.html [hunk 3]
```diff
@@ -0,0 +1 @@
+<button xotbButton variant="base">small button</button>
```
Comment (akuntal) [inline projects/xotb-demo/src/app/examples/buttons/buttons.component.html]: Add demo for other varian as well

File: projects/buttons/src/lib/button.ts (hunk 1 above)
Comment (yelendisureshbabu-hcl) [reply projects/buttons/src/lib/button.ts]: created button.html

File: projects/buttons/src/public-api.ts (hunk 2 above)
Comment (yelendisureshbabu-hcl) [reply projects/buttons/src/public-api.ts]: remove exporting of button component.

File: projects/xotb-demo/src/app/examples/buttons/buttons.component.html (hunk 3 above)
Comment (yelendisureshbabu-hcl) [reply projects/xotb-demo/src/app/examples/buttons/buttons.component.html]: added

**Pull request merged**